resources/rules.k.tmp

```diff
@@ -188,12 +188,16 @@ rule ACCTCODE in SetItem( 1 )
                  SetItem ( 6 )
                  SetItem ( 7 )
                  SetItem ( 8 )
+                 SetItem ( 9 )
                  => false
-     requires 9 <=Int ACCTCODE
+     requires ACCTCODE ==Int 0 orBool 10 <=Int ACCTCODE
 
     syntax IntList ::= bytesToWords ( WordStack )       [function]
 
- // --------------------------------------------------------------
+    rule BS [0 .. M] [0 .. N ] => BS [ 0 .. N]
```

Comment:
what is this for?

Reply:
@xwvvvvwx do you mean when is it used, or what does it do? IDK about the former, but the latter is: this reduces two subsequent accesses to the same ByteStack to one access, as long as the second access is contained within the first.